modules/BiotypeQC.nf

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nextflow
+
+/*
+BioQC is a is a R/Bioconductor package to detect tissue heterogeneity in gene expression data
+*/
+
+params.gene_expression = "/home/ausserh/myScratch/Salmon_test.txt"
+params.samplesheet = "/home/ausserh/testMatrix.tsv"
+params.marker = "extdata/exp.tissuemark.affy.roche.symbols.gmt"
+
+
+process 'BioQC' {
+
+  conda 'bioconductor-bioqc r-readr'
+  publishDir "/home/ausserh/BioQC_test", mode: 'copy'
+
+  input:
+  path gene_expression from params.gene_expression
+  path samplesheet from params.samplesheet
+  val marker from params.marker
+    
+  output:
+  path ('*.jpg')
```

Comment:
```suggestion
  path ('*.jpg'), emit: heatmap
```

Using named output channels is a good practice

Reply:
If you know what output file to expect I'd prefer to use the exact filename instead of globbing